sequence_classification/results_presenter.py

```diff
@@ -80,3 +84,52 @@ def show_box_plots(self):
     @staticmethod
     def calc_accuracy_from_cm(cm):
         return cm.diagonal().sum() / cm.sum()
+
+    def prepare_measurements(self):
+        if self.measurements:
+            return
+        self.measurements = []
+        for d in self.results:
+            self.measurements.append([])
+            for classifier_name, rounds in self.results[d].items():
+                self.measurements[-1].append([])
+                for r in rounds:
+                    acc = self.calc_accuracy_from_cm(r['conf_matrix_test'])
+                    self.measurements[-1][-1].append(acc)
+    
+    def calc_friedman_value(self):
+        self.prepare_measurements()
+        avg = []
+        for dataset in self.measurements:
+            avg.append([])
+            for classifier in dataset:
+                avg[-1].append(np.average(classifier))
+        return friedmanchisquare(*avg)
+
+    def calc_anova_value(self):
+        self.prepare_measurements()
+        anova = []
+        for dataset in self.measurements:
+            anova.append(f_oneway(*dataset))
+        return anova
+
+    def calc_nemenyi(self):
+        self.prepare_measurements()
+        avg = []
+        for dataset in self.measurements:
```

Comment:
Stwórz metodę, żeby nie kopiować kodu.
```Python
def calc_averages(self):
    avg = []
    for dataset in self.measurements:
        avg.append([])
        for classifier in dataset:
            avg[-1].append(np.average(classifier))
    return avg
```
Poza tym, coś mi świta, że `np.average()` mogłoby to wszystko zrobić przy odpowiednim przekazniu argumentu `axis`.